packages/terra-application-links/src/tabs/orion-fusion-theme/ApplicationTabs.module.scss

```diff
@@ -0,0 +1,80 @@
+:local {
+  .orion-fusion-theme {
```

Comment:
In https://github.com/cerner/terra-framework/blob/master/packages/terra-application-links/src/tabs/ApplicationTabs.module.scss there are 68 theme vars, but here there are 65. Are there any missing?

Reply:
Actually a couple of variables are used twice, once with calc function like [here](https://github.com/cerner/terra-framework/blob/b123efc4132de8e1b2235bd3e6a0e7dbbf4ae032/packages/terra-application-links/src/tabs/ApplicationTabs.module.scss#L22). No variables are missing